src/ng/components/body/body.html

```diff
@@ -1,7 +1,11 @@
-<tr ng-repeat="$row in $view.body.rows track by $index"
-	 ng-class="{'q-grid-selected': $view.selection.state($row).isSelected}"
-	 q-grid-core:tr>
+<tr ng-repeat-start="$row in $view.body.rows track by $index"
```

Comment:
Also we decided to not use ng-class in markup for the rows, columns and cells for perfomance.
Also view.body.columns can be done as function that accept row and now about colspan?
```html
<tr ng-repeat="$row in $view.body.rows track by $index">
	<td ng-repeat="$column in $view.body.columns($row) track by $column.model.key"
	       q-grid-core:td>
	</td>
</tr>
```